src/components/PersonList/PersonList.tsx

```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  person: Person,
+  selectedPerson: string,
+};
+
+export const PersonList: React.FC<Props> = ({
+  person,
+  selectedPerson,
+}) => {
+  const isSelected = selectedPerson === person.slug;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': isSelected,
+      })}
+    >
+      <td>
+        <PersonLink
+          to={person.slug}
+          name={person.name}
+          sex={person.sex}
+        />
+      </td>
+
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died}</td>
+      <td>
+        {person.mother
+          ? (
+            <PersonLink
+              to={person.mother?.slug}
+              name={person.motherName}
+              sex={person.mother.sex}
+            />
+          ) : (
+            person.motherName || '-'
```

Comment:
Will there be a mother's name if there is no mother?

Reply:
Yes, the motherName can be without the object mother.
![image](https://user-images.githubusercontent.com/55395284/198898031-1f7f206e-b6e0-4ff2-a248-976073d8d21a.png)